templates/commands/goldentest/test_funcs.go

```diff
@@ -212,3 +213,22 @@ func mapToVarValues(m map[string]string) []*goldentest.VarValue {
 	}
 	return out
 }
+
+func renameGitignoreFiles(dir string) error {
```

Comment:
I think we should support more than just `.gitignore` here. Maybe we should handle any file or directory beginning with `.git*`? That would support `.gitattributes` and it would support outputting a `.git` directory if anybody wants to do that.

Reply:
done

Reply:
I made `.github` an exception. I don't think we should rename `.github`

Reply:
I think we SHOULD rename .github. If there's a template that outputs a .github directory, then that .github directory should not affect how the golden data directory is handled by github. Or maybe I'm missing your point.

Reply:
Updated.